apps/multiplatform/common/src/commonMain/kotlin/chat/simplex/common/model/SimpleXAPI.kt

```diff
@@ -411,6 +413,23 @@ object ChatController {
 
   fun hasChatCtrl() = ctrl != -1L && ctrl != null
 
+  suspend fun getAgentServersSummary(rh: Long?): PresentedServersSummary {
+    val userId = currentUserId("getAgentServersSummary")
+
+    val r = sendCmd(rh, CC.GetAgentServersSummary(userId))
+
+    if (r is CR.AgentServersSummary) {
+      return r.serversSummary;
+    } else {
+      Log.e(TAG, "getAgentServersSummary bad response: ${r.responseType} ${r.details}")
+      throw Exception("failed to get agent servers summary ${r.responseType} ${r.details}")
+    }
+  }
+
+  suspend fun resetAgentServersStats() {
+    sendCommandOkResp(null, CC.ResetAgentServersStats())
```

Comment:
I think this actually requires remote host parameter (`rh`) to differentiate resetting stats for host/controller. i.e. if you reset when using mobile from desktop it would be expected to reset for mobile